app/views/directives/osc-source-secrets.html

```diff
@@ -5,7 +5,7 @@
         <div class="row advanced-secrets">
           <div class="col-lg-6">
             <label class="picker-label" ng-if="$first">Build Secret</label>
-            <ui-select ng-required="pickedSecret.destinationDir" ng-model="pickedSecret.secret.name">
+            <ui-select ng-model="pickedSecret.secret.name">
```

Comment:
Don't we still want to make the secret name required if a destination dir is entered? It doesn't make sense to entered a destination dir if you don't pick a secret.